### DISCUSSION TITLE
rclone support

### DISCUSSION THREAD

**Discussion by ghost** [javinizer/Javinizer#222] [Ideas]
Thanks you very much! This project saved my time!

I store all my collection in google drive. After sorting the videos with javinizer, I copy them to the google drive with rclone. So I never have to worry about sorting collections in the cloud. 

But I just realized that if I want to re-sort them in the future, my only option would be downloading them and sort in local drive. Then upload them back to the drive. But this solution is not feasible in my situation. Because my collection is quite big. Also, it would be much more reliable and efficient sort with rclone as google drive supports server side copy.

Is there any way I can config to use rclone instead of move command

**Comment by jvlflame** [comment]
I'm not sure if rclone is truly necessary for Google Drive sorting purposes. If you mount your GDrive on your local system, wouldn't you be able to manage it that way?

I believe I've had users in the past have success with mounting and sorting via GDrive mounted by [NetDrive](https://www.netdrive.net/). I'd say give that or any other similar solutions a shot.

**Reply by ghost** [reply]
Yes. But i am worried about data integrity. 

I believe move-item would not verity the transfers data. Sorting fils within drive would not be a problem, as windows simply change the path of files.

But for move files across drive, windows actually move the data and there is a chance for data corruption, especially across the network.

Nevermind. By that time i could simply perform a snapshot backup and check the files afterward.

**Comment by raenoll** [comment]
I use rclone as well. You need a vps instead of your local computer and it’s easier to operate.
